CHANGELOG.md

```diff
@@ -2,6 +2,49 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.3.0](https://github.com/rhobs/monitoring-stack-operator/commit/) (2025-11-03)
+
+
+### Features
+
+* add cluster-health-analyzer v0.5.1 ([#785](https://github.com/rhobs/monitoring-stack-operator/issues/785)) ([315b436](https://github.com/rhobs/monitoring-stack-operator/commit/315b43660968fe0e4a8e66e33a1fc4eb4643f9ea))
```

Comment:
1.3.0 is with cha v1.0.0 please

Reply:
hmm this is auto generated from commits

Reply:
Is this the GA notice? https://github.com/rhobs/observability-operator/pull/920/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR42